plugins/MultiSites/vue/src/AllWebsitesDashboard/AllWebsitesDashboard.vue

```diff
@@ -0,0 +1,194 @@
+<!--
+  Matomo - free/libre analytics platform
+
+  @link    https://matomo.org
+  @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+  <div class="dashboardHeader">
+    <h1 class="card-title">
+      <EnrichedHeadline
+          :feature-name="translate('MultiSites_AllWebsitesDashboardTitle')"
+      >
+        {{ translate('MultiSites_AllWebsitesDashboardTitle') }}
+      </EnrichedHeadline>
+    </h1>
+
+    <div v-if="!isWidgetized"
+         id="periodString"
+         class="borderedControl"
+    >
+      <PeriodSelector :periods="selectablePeriods" />
+    </div>
+  </div>
+
+  <KPICardContainer
+      :is-loading="isLoadingKPIs"
+      :model-value="kpis"
+  />
+
+  <div class="dashboardControls">
+    <div class="siteSearch">
+      <input
+          type="text"
+          @keydown.enter="searchSite(searchTerm)"
```

Comment:
@sgiehl good observation, I'm wondering why wouldn't we just do a debounce on text input (it's actually what I tried initially when testing).

Reply:
Can add the clear icon to the list of "things to check for final design".

With the search icon being on the other side compared to the usual search inputs it might not be good to just add it next to the icon itself, without moving the search icon also. (Though the icon placement currently matches the existing dashboard, but that is more an excuse than a reason)

For the blur event it currently is at least consistent with the other data table search fields. Debouncing and "instant search" is most likely only a safe option if we decide to handle all sorting and formatting in the UI instead of the backend. Using `(value).toLocaleString(Matomo.language, ...)` we could talk care of the display part if the amount of data can be transferred in one chunk (like ~10 values per site if we remove every excess information?) :thinking: